openedx/features/edly_panel/tests.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+from django.test import TestCase
+from .models import EdlyActiveUser
+from django.contrib.auth.models import User
+from django.utils import timezone
+
+
+class EdlyActiveUserModelTest(TestCase):
+
+    def setUp(self):
+        self.user = User.objects.create(username='test@example.com', password='test_password')
```

Comment:
Create factories in `edly_panel/tests/factories.py` using FactoryBoy and use them here instead of created object via model classes.
https://factoryboy.readthedocs.io/en/latest/orms.html

Reply:
done.